src/ResourceManager/Compute/Commands.Compute.Test/ScenarioTests/VirtualMachineExtensionTests.cs

```diff
@@ -122,6 +122,13 @@ public void TestAzureDiskEncryptionExtensionSinglePass()
             ComputeTestController.NewInstance.RunPsTest("Test-AzureDiskEncryptionExtensionSinglePass");
         }
 
+        [Fact(Skip = "TODO: only works for live mode due to key vault dependency")]
```

Comment:
@ejarvi just for our understanding, is there a reason that we can't create a KeyVault client in the test controller that allows you to use the KeyVault cmdlets in playback/record mode?

Reply:
Yes thanks for asking.  I would really like to get this enabled.  Once I have three different tests that I should be able to flip multiple tests to check-ins.  I spent time last night trying to get this to work locally yesterday but was unsuccessful.   After updating the package references to use the 4.2.1-alpha version of Key Vault so that it could find dependency references, I got hung up on an exception indicating the KeyVaultClient constructor was not being called in the correct format.  This was from within test controller code I'm not yet familiar with and apparently across a package boundary. If I recall correctly the exception surfaced from Microsoft.Azure.Common.Compute.Authentication.dll.   I haven't had time to drill deeper yet.

Reply:
taking another stab at this now...

Reply:
@ejarvi okay thanks for the information. @maddieclayton and I can work with you in the next sprint to mitigate this issue, and we should be able to use the Service Fabric tests as an example since they are currently using KeyVault cmdlets in their tests